src/mysky/utils.ts

```diff
@@ -0,0 +1,67 @@
+import { ensureUrl } from "skynet-interface-utils";
+
+import { SkynetClient } from "../client";
+import { getFullDomainUrlForPortal, extractDomainForPortal } from "../utils/url";
+
+/**
+ * Constructs the full URL for the given component domain,
+ * e.g. "dac.hns" => "https://dac.hns.siasky.net"
+ *
+ * @param this - SkynetClient
+ * @param domain - Component domain.
+ * @returns - The full URL for the component.
+ */
+export async function getFullDomainUrl(this: SkynetClient, domain: string): Promise<string> {
+  const portalUrl = await this.portalUrl;
+
+  return getFullDomainUrlForPortal(portalUrl, domain);
+}
+
+export async function extractDomain(this: SkynetClient, fullUrl: string): Promise<string> {
```

Comment:
missing docstring

Reply:
I know, I can add them later. It's not pressing.